### PR TITLE
Add toLowerCase() to search input

### LINKED ISSUE #138: Make search input lowercase
## Expected Behavior
Should be able to search with uppercase or capitalized names.

## Actual/Current Behavior
Search does not work if not all lowercase.

## Steps to Reproduce the Problem
  1. Open app.
  2. Search pokemon name via search bar with capital letter.
  3. Observe.

## Pre-Testing TODOs

_What needs to be done before testing._

  1. Add `toLowerCase` to search bar input value before trim.

### If you are not a member of this project, _skip this step_

  1. Open app.
  2. Search pokemon name with uppercased letters.
  3. Observe

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Make search input value lowercase.

## Purpose
If user searches in all uppercase or with capitalized name, search won't work. 

## Approach
Add this method to make value all lowercase for searching.

Closes #138

### CODE REVIEW

File: src/components/PokeApp.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const PokeApp = () => {
   const [isLoading, setIsLoading] = useState(true);
   // search bar onChange function for searching pokemon & conditional rendering
   const handleNameSearch = (e) => {
-    const searchBarValue = e.target.value.trim();
+    const searchBarValue = e.target.value.toLowerCase().trim();
```
Comment (dannysanchez559) [inline src/components/PokeApp.js:21]: this takes care of the search issue nicely!

**Review by dannysanchez559 — APPROVED** [review]
Nice

**Review by BlackCubes — APPROVED** [review]
Nice and simple fix!

**Review by JorgeLVilla — APPROVED** [review]
Nice job Connie!

**Pull request merged**